includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -130,6 +130,16 @@ protected function should_show_facebook() {
 		return current_user_can( 'install_plugins' );
 	}
 
+	/**
+	 * Is the WooCommerce Admin actively included in the WooCommerce core?
+	 * Based on presence of a basic WC Admin function.
+	 *
+	 * @return boolean
+	 */
+	protected function is_wc_admin_included_in_wc() {
```

Comment:
This method name doesn't sounds correct, because it will check if WooCommerce Admin is "activated" no matter if is included in WooCommerce or not.
Also probably better follow WordPress or other WooCommerce examples for this method name, like `wc_admin_exists()` or `is_wc_admin_installed()`.

Reply:
Agree. Updated to `is_wc_admin_active()`...